nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessPropagation.java

```diff
@@ -1085,6 +1086,13 @@ public TransferResult<Nullness, NullnessStore> visitExpressionStatement(
     return noStoreChanges(NULLABLE, input);
   }
 
+  @Override
+  public TransferResult<Nullness, NullnessStore> visitDeconstructorPattern(
+      DeconstructorPatternNode deconstructorPatternNode,
+      TransferInput<Nullness, NullnessStore> input) {
+    return noStoreChanges(NULLABLE, input);
```

Comment:
Why is this marked as not covered? Because the codecov report is for an older JDK?

Reply:
It's due to https://github.com/typetools/checker-framework/issues/6215, which is now fixed.  Before, these nodes were not getting added properly to CFGs by Checker dataflow.  Turns out checking code coverage is handy!

Reply:
We won't get the coverage until Checker Framework does another release and we update, but I don't think it's worth waiting for that to land this.